src/legacy/core_plugins/console/np_ready/public/application/components/console_menu.tsx

```diff
@@ -141,17 +174,26 @@ export class ConsoleMenu extends Component<Props, State> {
     ];
 
     return (
-      <span onMouseEnter={this.mouseEnter}>
-        <EuiPopover
-          id="contextMenu"
-          button={button}
-          isOpen={this.state.isPopoverOpen}
-          closePopover={this.closePopover}
-          panelPaddingSize="none"
-          anchorPosition="downLeft"
-        >
-          <EuiContextMenuPanel items={items} />
-        </EuiPopover>
+      <span>
+        <span onMouseEnter={this.mouseEnter}>
+          <EuiPopover
+            id="contextMenu"
+            button={button}
+            isOpen={this.state.isPopoverOpen}
+            closePopover={this.closePopover}
+            panelPaddingSize="none"
+            anchorPosition="downLeft"
+          >
+            <EuiContextMenuPanel items={items} />
+          </EuiPopover>
+        </span>
+        <span>
+          <EuiGlobalToastList
```

Comment:
Kibana publishes a global toasts service we should use instead. That way any toasts added by another part of the UI will stack with the toasts added here. Here's an example of the service, called `notifications`, accessed via the New Platform: https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/remote_clusters/public/shim.ts#L23. You can grep the app to see examples of its consumption.